app/views/shared/_gtag.html.haml

```diff
@@ -1,10 +0,0 @@
-%script{async: true,  src: "https://www.googletagmanager.com/gtag/js?id=#{ENV['GOOGLE_ANALYTICS']}"}
-
-:javascript
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-  gtag('config', "#{ENV['GOOGLE_ANALYTICS']}", { 'optimize_id': "#{ENV['GOOGLE_OPTIMIZE_ID']}", 'anonymize_ip': true, 'linker': {
```

Comment:
Will Google Optimize continue to function?
Will the "accept_incoming" continue to function?
Will ips continue to be anonymized?
If we're removing these environment variables from there are they used elsewhere?  Can we remove them entirely from the app including terraform etc?

Reply:
Google optimize will continue to function
🤷‍♂ 
Yes, this was configured in Tag Manager
They're not used elsewhere so you're right that we should remove them from Terraform

Reply:
The accept_incoming has been set up on google tag manager

Reply:
Terraform vars removed in https://github.com/DFE-Digital/teacher-vacancy-service/pull/1032/commits/47198cf32e83bfb5b37c56280067f046aae5450a